code/include/swoc/Lexicon.h

```diff
@@ -135,8 +149,7 @@ template <typename E> class Lexicon {
    *
    * @see set_default.
    */
-  explicit Lexicon(const std::initializer_list<Definition> &items, DefaultHandler handler_1 = DefaultHandler{},
-                   DefaultHandler handler_2 = DefaultHandler{});
+  explicit Lexicon(with_multi items, Default handler_1 = Default{}, Default handler_2 = Default{});
```

Comment:
Does this need to be explicit?

Reply:
It shouldn't be, since it's not single argument.